docs/src/components/ExternalExampleLayout.tsx

```diff
@@ -49,34 +53,53 @@ class ExternalExampleLayout extends React.Component<
     })
 
     if (!examplePath) return <PageNotFound />
+
     const exampleSource: ExampleSource = exampleSourcesContext(examplePath)
-    const isRtlEnabled = rtl === 'true'
+    const theme = this.getTheme()
 
     return (
-      <Provider key={this.state.renderId} theme={{ rtl: isRtlEnabled }}>
-        <div dir={isRtlEnabled ? 'rtl' : undefined}>
-          <SourceRender
-            babelConfig={babelConfig}
-            source={exampleSource.js}
-            renderHtml={false}
-            resolver={importResolver}
-          >
-            <SourceRender.Consumer>
-              {({ element, error }) => (
-                <>
-                  {element}
-                  {/* This block allows to see issues with examples as visual regressions. */}
-                  {error && (
-                    <div style={{ fontSize: '5rem', color: 'red' }}>{error.toString()}</div>
+      <Provider key={this.state.renderId} theme={theme}>
+        <Provider.Consumer
+          render={({ siteVariables }) => (
+            <div
+              dir={theme.rtl ? 'rtl' : undefined}
+              style={{
```

Comment:
Unfortunately this is needed in order to see the actual changes in other themes, and that's because `color` and `backgroundColor` are not set for a lot of components in other themes.

However, this means the styles are changes for all screenshots so we will see changes in all of them...

Reply:
These changes will be done in #852, as you can see it has the same regressions 💣

Reply:
I propose, to add them right now, but before this PR. What do you think?

Reply:
I propose, to add them right now, but before this PR. 

@layershifter not sure I understand, you propose to merge #852 before this or? who is `them` in this context? 😆

Reply:
got it, you mean to remove the changes and wait for #852 wel..the tests will be useless until then, I don't really like that especially that the #852 is not that active anymore

Reply:
agreed to split this in 2 PRs to avoid screen regressions here, the new PR will depend on #852

Reply:
resolving, tracked in newly opened #1050